packages/design-system/src/components/Tabs/TabPanel.tsx

```diff
@@ -41,6 +41,9 @@ export const TabPanel = (props: TabPanelProps) => {
   const classes = classnames('ds-c-tabs__panel', props.className);
 
   return (
+    // 🥑
+    // https://www.w3.org/TR/wai-aria-1.2/#tabpanel
+    //    - "aria-disabled (state) (deprecated on this role in ARIA 1.2)"
     <div
```

Comment:
TODO: Explore purpose of disabled Tabs. It's recommended not to disable interactive elements as it can confuse users. 

`aria-disabled` is deprecated on `tabpanel` roles. Need to figure out purpose of disabling component before mitigation can be decided.